[2024-02-26T09:37:24.413+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-26T09:37:24.435+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [queued]>
[2024-02-26T09:37:24.477+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [queued]>
[2024-02-26T09:37:24.477+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2024-02-26T09:37:24.525+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_pet_table> on 2020-02-02 00:00:00+00:00
[2024-02-26T09:37:24.574+0000] {standard_task_runner.py:57} INFO - Started process 4444 to run task
[2024-02-26T09:37:24.591+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', '***_operator_dag', 'create_pet_table', 'scheduled__2020-02-02T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/etl_***.py', '--cfg-path', '/tmp/tmpbb2_wljk']
[2024-02-26T09:37:24.599+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask create_pet_table
[2024-02-26T09:37:26.117+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-26T09:37:26.169+0000] {task_command.py:410} INFO - Running <TaskInstance: postgres_operator_dag.create_pet_table scheduled__2020-02-02T00:00:00+00:00 [running]> on host 1db92e94aa46
[2024-02-26T09:37:26.201+0000] {logging_mixin.py:150} WARNING - /home/airflow/.local/lib/python3.7/site-packages/airflow/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-02-26T09:37:26.231+0000] {abstractoperator.py:598} ERROR - Exception rendering Jinja template for task 'create_pet_table', field 'sql'. Template: 'sql/pet/list_pet.sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 153, in render_template
    template = jinja_env.get_template(value)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 214, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sql/pet/list_pet.sql
[2024-02-26T09:37:26.235+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1407, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1531, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2179, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/baseoperator.py", line 1254, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 76, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/abstractoperator.py", line 590, in _do_render_template_fields
    seen_oids,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/template/templater.py", line 153, in render_template
    template = jinja_env.get_template(value)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 125, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/airflow/.local/lib/python3.7/site-packages/jinja2/loaders.py", line 214, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: sql/pet/list_pet.sql
[2024-02-26T09:37:26.244+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=***_operator_dag, task_id=create_pet_table, execution_date=20200202T000000, start_date=20240226T093724, end_date=20240226T093726
[2024-02-26T09:37:26.252+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 13 for task create_pet_table (sql/pet/list_pet.sql; 4444)
[2024-02-26T09:37:26.294+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2024-02-26T09:37:26.496+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
